tests/migration/Core/V6_4/Migration1665267882RenameCountryVatTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Shopware\Tests\Migration\Core\V6_4;
+
+use Doctrine\DBAL\Connection;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelTestBehaviour;
+use Shopware\Core\Migration\V6_4\Migration1665267882RenameCountryVat;
+
+class Migration1665267882RenameCountryVatTest extends TestCase
+{
+    use KernelTestBehaviour;
+
+    public function testChanges() : void
+    {
+        /** @var Connection $connection */
+        $connection = $this->getContainer()->get(Connection::class);
+
+        $sql = 'SELECT language.name, country_translation.name FROM country
+LEFT JOIN country_translation ON country.id = country_translation.country_id
+LEFT JOIN language ON language.id = country_translation.language_id
+WHERE country.iso = :iso AND country.iso3 = :iso3';
+
+        $translations = $connection->fetchAllKeyValue($sql, ['iso' => 'VA', 'iso3' => 'VAT']);
+        $this->assertEquals(
```

Comment:
Did you run this test? This assertion should fail as the tests run after all migrations are already executed, so you would have to revert your migration manually in the `setUp` of the test case

Reply:
I did run this test, but only manually. I have adjusted the test accordingly.